lightwood/mixer/regression.py

```diff
@@ -41,10 +57,24 @@ def fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
         log.info(f'Regression based correlation of: {self.model.score(X, Y)}')
 
     def partial_fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
+        """
+        Fits the Neural mixer on some data, this refits the model entirely rather than updating it
```

Comment:
Typo: 
> Fits the _regression_ mixer

Reply:
You need to actually discuss how partial-fit is handled in LinearRegression. LinearRegression is cheap, hence we simply re-fit the model with more data (old data + new data). This is NOT the same as in other models, for example, neural networks, where you can get away with training on just new data assuming the old data representations have been "learned" or fine-tuning.

Reply:
There is no choice to motivate. If you retrain Linear regression with a new dataset, it's as if the old dataset was never seen. 

The problem is the term `partial_fit` is ambiguous, so we need to be clear what `refit` means, because in certain contexts `refitting` means that we can train without the old data (i.e. neural nets can be fine-tuned with new data). Terms like `update` and `refit` do not discuss *how* changes are made. It doesn't hurt to be explicit.